.env.example

```diff
@@ -7,8 +7,8 @@ APP_URL=http://dev-battle-landing.dev/
 
 DB_CONNECTION=mysql
 DB_HOST=127.0.0.1
-DB_PORT=3306
-DB_DATABASE=dev-battle-landing
+DB_PORT=3307
```

Comment:
Let's keep this at `3306`, since it's only `3307` on your local. Thanks!

Reply:
Updated!